python_modules/dagster/dagster/_core/storage/compute_log_manager.py

```diff
@@ -90,11 +99,16 @@ class CapturedLogMetadata(
             ("stderr_location", Optional[str]),
             ("stdout_download_url", Optional[str]),
             ("stderr_download_url", Optional[str]),
+            ("log_manager_metadata", Optional[str]),
+            ("stdout_uri_or_path", Optional[str]),
+            ("stderr_uri_or_path", Optional[str]),
         ],
     )
 ):
-    """Object representing metadata info for the captured log data, containing a display string for
-    the location of the log data and a URL for direct download of the captured log data.
+    """Object representing metadata info for the captured log data.
+    It contains:
```

Comment:
More accurately _can_ contain; since it does not always contain this information